aws/table_aws_wafregional_web_acl.go

```diff
@@ -0,0 +1,305 @@
+package aws
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/wafregional"
+	"github.com/aws/aws-sdk-go-v2/service/wafregional/types"
+	wafregionalv1 "github.com/aws/aws-sdk-go/service/wafregional"
+	"github.com/aws/smithy-go"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsWafRegionalWebAcl(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_wafregional_web_acl",
+		Description: "AWS WAF Regional Web ACL",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"web_acl_id"}),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"WAFNonexistentItemException", "WAFInvalidParameterException"}),
+			},
+			Hydrate: getWafRegionalWebAcl,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listWafRegionalWebAcls,
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(wafregionalv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the Web ACL. You cannot change the name of a Web ACL after you create it.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the entity.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWafRegionalWebAcl,
+				Transform:   transform.FromField("WebACLArn"),
+			},
+			{
+				Name:        "web_acl_id",
+				Description: "The unique identifier for the Web ACL.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("WebACLId"),
+			},
+			{
+				Name:        "default_action",
+				Description: "The action to perform if none of the Rules contained in the WebACL match.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWafRegionalWebAcl,
+				Transform:   transform.FromField("DefaultAction.Type"),
+			},
+			{
+				Name:        "metric_name",
+				Description: "A friendly name or description for the metrics for this WebACL.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWafRegionalWebAcl,
+			},
+			{
+				Name:        "logging_configuration",
+				Description: "The logging configuration for the specified web ACL.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getWafRegionalLoggingConfiguration,
```

Comment:
```suggestion
				Hydrate:     getWafRegionalLoggingConfiguration,
				Transform: transform.FromValue(),
```

Reply:
They have a `LoggingConfiguration` field in the returned object